modules/tinymce/src/core/main/ts/api/EditorManager.ts

```diff
@@ -24,12 +24,15 @@ import URI from './util/URI';
 declare const window: Window & { tinymce: any; tinyMCEPreInit: any };
 
 /**
+ * NOTE: the class tag is commented out for the include in
+ * `modules/tinymce/tools/docs/tinymce.js`
+ *
```

Comment:
Can we move this note outside the `EditorManager` class comment block? eg:
```suggestion
// NOTE: the class tag is commented out for the include in `modules/tinymce/tools/docs/tinymce.js`
/**
```

Reply:
Fixed in [`b995397` (#6775)](https://github.com/tinymce/tinymce/pull/6775/commits/b99539748bc82c00e104582adc37a9b0db558e01)